core/lib/generators/solidus/install/app_templates/frontend/classic.rb

```diff
@@ -3,7 +3,10 @@
 if Bundler.locked_gems.dependencies['solidus_frontend']
   say_status :skipping, "solidus_frontend is already in the bundle", :blue
 else
-  bundle_command("add solidus_frontend")
+  # `solidus_frontend` is not sourced from rubygems if the solidus gem was not.
+  github_solidus_frontend = '--github solidusio/solidus_frontend' unless solidus.nil? || solidus.source.is_a?(Bundler::Source::Rubygems)
```

Comment:
nit: I think this can be reversed to `if solidus && !solidus.source.is_a?(Bundler::Source::Rubygems)`

Reply:
@elia I can submit a separate PR to make this refactoring. At the moment, I would just like to get this PR merged so that I can get the SolidusBraintree CI to pass...